dev-haskell/haskell-lsp/haskell-lsp-0.22.0.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
```

Comment:
Might be worth putting in another `#hackport: flags: demo:demo-exe` line here to preserve the flag rename.